test/t8n/t8n.cpp

```diff
@@ -128,6 +132,11 @@ int main(int argc, const char* argv[])
                 }
             }
 
+            if (block_reward.has_value())
+                state.touch(block.coinbase).balance += *block_reward;
+            else if (rev <= EVMC_TANGERINE_WHISTLE)  // This behaviour is required by retesteth
+                state.get_accounts().erase(block.coinbase);
```

Comment:
Hm, this will remove it even when other transactions touched this account. Is that desired?

Reply:
Sorry. Fore some reason I remove also checking if it's empty. Fixed now

Reply:
I'm still not convinced by this quirk. It looks it replicates a bug in some client implementation. Can you provide a list of failing tests?